ipcalc.c

```diff
@@ -573,29 +573,6 @@ char *ipv6_prefix_to_hosts(char *hosts, unsigned hosts_size, unsigned prefix)
 	return hosts;
 }
```

Comment:
I find flags quite easier to handle and manipulate. That is, you can mask multiple easier, and it is harder to forget unused variables around. The whole discussion is bikeshedding though, one way or another is fine. Unless you really have a good argument, I'd keep the existing way for no particular reason.

r-